packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/modals/CreateQuestionModal.tsx

```diff
@@ -202,13 +213,16 @@ const CreateQuestionModal: React.FC<CreateQuestionModalProps> = ({
         />
       </Form.Item>
 
-      {/* TODO: change this to only be an option if the queue is hybrid. Strictly in-person or online queues should not have this option */}
-      <Form.Item name="location" label="Are you joining the queue in-person?">
-        <Radio.Group className="mb-1">
-          <Radio value="In Person">Yes</Radio>
-          <Radio value="Online">No</Radio>
-        </Radio.Group>
-      </Form.Item>
+      {isQueueHybrid && (
+        <Form.Item name="location" label="Is the question in-person?">
```

Comment:
Similarly with AddStudentsToQueueModal, you will want to add the following to the `<Form>` element, otherwise location will be null if the user doesn't click anything
```
initialValues={{
            location: isQueueHybrid ? 'Online' : undefined,
          }}
```
![image](https://github.com/user-attachments/assets/f6fca718-8309-48d1-8926-bd0c5ab870ff)

Reply:
I missed this, thanks!